### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by yemidada**
In this milestone, I
- Use React Router.
- Use styles in a React app.

### CODE REVIEW

File: src/components/Calculator/Buttons.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (omarsalem7) [inline src/components/Calculator/Buttons.jsx]: - [ ] Kindly consider avoiding disabling linters can identify common coding mistakes, potential bugs, and errors early in the development process. They analyze the code for syntax errors, unused variables, improper variable assignments, and other issues that might cause problems during runtime. Detecting and fixing these errors early saves time and effort in the debugging phase.

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,26 @@
-import Calcultor from './components/Calculator';
+import { Suspense } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (omarsalem7) [inline src/App.js]: - [ ] As stated before, kindly consider avoiding disable linters

File: src/components/Calculator/CalculatorScreen.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (omarsalem7) [inline src/components/Calculator/CalculatorScreen.jsx]: - [ ] As stated before, kindly consider avoiding disable linters

File: src/components/Navigation/Navigation.jsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (omarsalem7) [inline src/components/Navigation/Navigation.jsx]: - [ ] As stated before, kindly consider avoiding disable linters

File: src/components/Quote/QuoteList.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (omarsalem7) [inline src/components/Quote/QuoteList.jsx]: - [ ] As stated before, kindly consider avoiding disable linters

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @yemidada 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights

- Use router for navigation ✔️
- Good UI ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @yemidada, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**